src/translations/en.json

```
@@ -3508,7 +3508,11 @@
               "instance_will_be_available": "Your instance will be available at your",
               "link_learn_how_it_works": "Learn how it works",
               "nabu_casa_url": "Nabu Casa URL",
-              "certificate_info": "Certificate info"
+              "external_activation": "Allow external activation",
+              "external_activation_secondary": "Should you be able to turn on remote control from your Nabu Casa account page.",
```

```suggestion
              "external_activation_secondary": "Allows you to turn on remote control from your Nabu Casa account page, even if you're outside your local network.",
```
would it be worth mentioning that this works if I'm outside my network? I guess that's the whole point of it, but I'm not sure how obvious it is.

---

Went for your suggestion, looks good